### PR TITLE
Fix proxy template for bottlerocket bootstrap

### PR DESCRIPTION
**Pull request by mrajashree**
*Issue #, if available:*

*Description of changes:*
Bottlerocket expects no-proxy setting to be a comma-separated list
of strings. The proxy template was parsing the input no-proxy list
and converting it to a string. This commit changes the template to
get the desired format.

Current generated value of no-proxy: "[localhost 127.0.0.1]"
New generated value for no-proxy: ["localhost", "127.0.0.1"]

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/65#" title="Author self-approved">mrajashree</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere-build-tooling).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere-build-tooling/blob/main/OWNERS)~~ [mrajashree]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by abhinavmpandey08 — APPROVED** [review]

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/65#pullrequestreview-768361548" title="Approved">abhinavmpandey08</a>*, *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/65#" title="Author self-approved">mrajashree</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere-build-tooling).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere-build-tooling/blob/main/OWNERS)~~ [abhinavmpandey08,mrajashree]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
